packages/react-vapor/src/components/table-hoc/TableRowNumberColumn.tsx

```diff
@@ -1,13 +1,8 @@
-import classNames from 'classnames';
 import * as React from 'react';
 
-import * as styles from './styles/TableRowNumber.scss';
-
 // eslint-disable-next-line id-blacklist
 export const TableRowNumberColumn = (props: {number: React.ReactNode}) => (
-    <td key="table-row-number" className={classNames('mod-border-right', styles.tableNumberedRow)}>
-        <div className={classNames('flex flex-column flex-center center', styles.tableNumberedRowContainer)}>
-            {props.number}
-        </div>
+    <td key="table-row-number" className={'table-numbered-row mod-border-right'}>
+        <div className={'table-numbered-row-container flex flex-column flex-center center'}>{props.number}</div>

```

Comment:
you don't need the curly brackets here if there''s no javascript in the className